client/src/components/Common/Heading.vue

```diff
@@ -33,18 +49,29 @@ const element = computed(() => {
 <template>
     <div v-if="props.separator" class="separator heading">
         <div class="stripe"></div>
-        <component :is="element" :class="[sizeClass, props.bold ? 'font-weight-bold' : '']">
+        <component
+            :is="element"
+            :class="[sizeClass, props.bold ? 'font-weight-bold' : '', collapsible ? 'collapsible' : '']"
+            @click="$emit('click')">
             <slot />
+            <FontAwesomeIcon v-if="collapsible" :icon="collapsed ? 'chevron-down' : 'chevron-up'" />
```

Comment:
Due to the icon being behind the slot, it will be in a different position for every heading. This makes quickly collapsing / opening multiple sections challenging. Moving it to the front would alleviate this. It would also improve seeing what sections are collapsed/open at a glance.

In the case of the separator variant, it maybe replace the first stripe, to be in line with the non-separator collapsible indicators.

Reply:
I actually had it to the left at the start, but the other icon (if provided) is on that side of the slot and it looked weird in my testing.  I'll tinker with a few options, though.